src/quo/components/list/item.cljs

```diff
@@ -5,12 +5,22 @@
             [quo.gesture-handler :as gh]
             [quo.design-system.spacing :as spacing]
             [quo.design-system.colors :as colors]
-            [quo.components.text :as text]
+            [clojure.string :as string]
             [quo.components.controls.view :as controls]
             [quo.components.tooltip :as tooltip]
+            [quo2.components.markdown.text :as text]
             [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as quo2.colors]
+            [quo2.components.icon :as quo2.icons]
```

Comment:
I believe any new components code should be in `quo2`.
Also, probably better to import `quo2.colors` as `colors`, and get rid of the old one (or change it to `quo.colors` if it must still be there for some reason)

Reply:
Thanks Omar, Done